cypress/integration/seededFlows/homeFeedFlows/feedNavigation.spec.js

```diff
@@ -14,43 +14,43 @@ describe('Home Feed Navigation', () => {
   it('should show Feed by default', () => {
     cy.findByRole('navigation', { name: 'View posts by' }).within(() => {
       cy.findByRole('link', { name: 'Feed' }).as('feed');
-      cy.findByRole('link', { name: 'Week' }).as('week');
-      cy.findByRole('link', { name: 'Month' }).as('month');
-      cy.findByRole('link', { name: 'Year' }).as('year');
-      cy.findByRole('link', { name: 'Infinity' }).as('infinity');
+      cy.findByRole('link', { name: 'Top' }).as('top');
+      cy.findByRole('link', { name: 'Latest' }).as('latest');
 
-      cy.findByRole('link', { name: 'Feed' }).should(
-        'have.attr',
-        'aria-current',
-        'page',
-      );
+      cy.get('@feed').should('have.attr', 'aria-current', 'page');
 
-      cy.get('@week').should('not.have.attr', 'aria-current');
-      cy.get('@month').should('not.have.attr', 'aria-current');
-      cy.get('@year').should('not.have.attr', 'aria-current');
-      cy.get('@infinity').should('not.have.attr', 'aria-current');
+      cy.get('@top').should('not.have.attr', 'aria-current');
+      cy.get('@latest').should('not.have.attr', 'aria-current');
     });
   });
 
-  it('should navigate to Week view', () => {
+  it('should navigate to Week view by default for Top', () => {
     cy.findByRole('navigation', { name: 'View posts by' }).within(() => {
-      cy.findByRole('link', { name: 'Week' }).as('week');
-      cy.get('@week').should('not.have.attr', 'aria-current');
-      cy.get('@week').click();
+      cy.findByRole('link', { name: 'Top' }).as('top');
+      cy.get('@top').click();
     });
 
     cy.url().should('contain', '/top/week');
     // Get a fresh handle to elements, as we've navigated to a new page
     cy.findByRole('navigation', { name: 'View posts by' }).within(() => {
-      cy.findByRole('link', { name: 'Week' }).should(
-        'have.attr',
-        'aria-current',
-        'page',
-      );
+      cy.findByRole('link', { name: 'Week' }).as('week');
+
+      cy.get('@week').should('have.attr', 'aria-current', 'page');
+
+      cy.get('@week').click(); // should not change the page
+      cy.url().should('contain', '/top/week'); // so the url stays the same
     });
   });
 
   it('should navigate to Month view', () => {
+    cy.findByRole('navigation', { name: 'View posts by' }).within(() => {
+      cy.findByRole('link', { name: 'Top' }).as('top');
+      cy.get('@top').click();
+    });
+
+    // this url check serves to wait for the page transition
+    cy.url().should('contain', '/top/week');
```

Comment:
if anyone feels up to making "find top, click top, wait for refresh" a function that would clean up a bit of boilerplate that ended up in all of the tests.